### PR TITLE
Fix for footnote Issues

### PR DESCRIPTION
**Pull request by Anand-wolverino**

### CODE REVIEW

File: document/static/js/modules/exporter/print/index.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ export class PrintExporter extends HTMLExporter {
             }
             section:footnote-content {
                 display: block;
+                font-style:normal;
+                font-weight:normal;
             }
```
Comment (johanneswilm) [inline document/static/js/modules/exporter/print/index.js]: could you also add text-decoration? I assume this is also true for underline

Comment (regostar) [reply document/static/js/modules/exporter/print/index.js]: but do we have underline option in fidus?

Comment (johanneswilm) [reply document/static/js/modules/exporter/print/index.js]: Yes. If you enable it in the document template designer.

Comment (Anand-wolverino) [reply document/static/js/modules/exporter/print/index.js]: Okay, sure I will make the necessary changes

**Pull request merged**